cmd/lotus-shed/export-car.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
+	"context"
```

Comment:
CI says this is unused

Reply:
"just one little change, what could go wrong"